java/arcs/core/host/ArcHostContext.arcs

```diff
@@ -22,14 +22,14 @@ schema Particle
   handles: [&HandleConnection]
 
 particle ArcHostContextParticle
-  arcHostContext: reads writes ArcHostContext {
+  arcHostContext: reads writes ArcHostContextHandle {
```

Comment:
This is definitely not a handle... maybe a different approach would be to change the package name of the generated code to `arcs.core.host.generated` ?

Reply:
Good idea!

Reply:
Updated!